eng/Packages.Data.props

```diff
@@ -108,6 +108,7 @@
     <PackageReference Update="Azure.Communication.Identity" Version="1.3.1" />
     <PackageReference Update="Azure.Communication.Common" Version="1.3.0" />
     <PackageReference Update="Azure.Core" Version="1.42.0" />
+    <PackageReference Update="Azure.Identity" Version="1.12.0" />
```

Comment:
This should not be needed here.   Azure SDK libraries do not take a direct dependency on Azure.Identity.   Please explain.